src/gt4py/backend/gt_backends.py

```diff
@@ -18,23 +18,24 @@
 import functools
 import numbers
 import os
-import types
-from typing import Optional
+from typing import Any, Dict, List, Tuple, Type, Union
```

Comment:
```suggestion
from typing import Any, Dict, List, Optional, Tuple, Type, Union
```

Reply:
`typing.Optional` is not used in this module anymore currently